docs/getting-started/config/authentication.md

```diff
@@ -8,43 +8,48 @@ Audience: Admins or Developers
+There are multiple authentication providers available for you to use with Backstage, feel free to follow [the instructions for adding authentication](../../auth/index.md).
 
-For this tutorial we choose to use GitHub, a free service most of you might be
-familiar with. For other options, see
+For this tutorial we choose to use GitHub, a free service most of you might be familiar with. For other options, see
 [the auth provider documentation](../../auth/github/provider.md#create-an-oauth-app-on-github).
 
-Go to
-[https://github.com/settings/applications/new](https://github.com/settings/applications/new)
-to create your OAuth App. The `Homepage URL` should point to Backstage's
-frontend, in our tutorial it would be `http://localhost:3000`. The
-`Authorization callback URL` will point to the auth backend, which will most
-likely be `http://localhost:7007/api/auth/github/handler/frame`.
+Go to [https://github.com/settings/applications/new](https://github.com/settings/applications/new) to create your OAuth App. The `Homepage URL` should point to Backstage's frontend, in our tutorial it would be `http://localhost:3000`. The `Authorization callback URL` will point to the auth backend, which will most likely be `http://localhost:7007/api/auth/github/handler/frame`.
 
 ![Screenshot of the GitHub OAuth creation page](../../assets/getting-started/gh-oauth.png)
 
-Take note of the `Client ID` and the `Client Secret`. Open `app-config.yaml`,
-and add your `clientId` and `clientSecret` to this file. It should end up
-looking like this:
+Take note of the `Client ID` and the `Client Secret` (clicking the "Generate a new client secret" button will get this value for you). Open `app-config.yaml`, and add your `clientId` and `clientSecret` to this file. It should end up looking like this:
 
 ```yaml title="app-config.yaml"
 auth:
   # see https://backstage.io/docs/auth/ to learn about auth providers
+  /* highlight-add-start */
   environment: development
+  /* highlight-add-end */
   providers:
+    # See https://backstage.io/docs/auth/guest/provider
+    guest: {}
+    /* highlight-add-start */
     github:
       development:
         clientId: YOUR CLIENT ID
         clientSecret: YOUR CLIENT SECRET
+        signIn:
+          resolvers:
+            # Matches the GitHub username with the Backstage user entity name.
+            # See https://backstage.io/docs/auth/github/provider#resolvers for more resolvers.
+            - resolver: usernameMatchingUserEntityName
+    /* highlight-add-end */
 ```
 
+:::note Note
+
+The default Backstage app comes with a guest Sign In Resolver. This resolver makes all users share a single "guest" identity and is only intended as a minimum requirement to quickly get up and running. You can read more about how [Sign In Resolvers](../../auth/identity-resolver.md#sign-in-resolvers) play a role in creating a [Backstage User Identity](../../auth/identity-resolver.md#backstage-user-identity) for logged in users.
+
+:::
```

Comment:
Let's move this up top, seems like an important detail before diving into the full guide.

Reply:
Sure, done!